### DISCUSSION TITLE
Open-source Git client for macOS minimalists

### DISCUSSION THREAD

**Discussion by torarnv** [tailwindlabs/tailwindcss#12703] [General]
What's the source of this website? https://commit.tailwindui.com/

Where can I find this tool, either in binary form our source code? 

The screenshots seem to point to the author working on Tailwind CSS:

![image](https://github.com/tailwindlabs/tailwindcss/assets/7536/331c8a81-3a8a-462a-b706-83408c49b476)

But couldn't find any references to "Joe Davola" elsewhere, and the sign-up box is non-functioning.

Thanks! :)

**Comment by fordaaron** [comment]
It's not an actual tool, just a demo for the template.
https://twitter.com/adamwathan/status/1647262690207825921

**Comment by torarnv** [comment]
Thanks @fordaaron !

@adamwathan is there an actual app behind there somewhere, or is it all mock-ups? 😄

**Reply by x0bd** [reply]
Here's the (un)official app I found based on the template: https://github.com/m1guelpf/commit
